mirage/views/search.ts

```diff
@@ -74,14 +123,46 @@ const resourceMetadataByType: Partial<Record<OsfmapResourceTypes, any>> = {
             '@type': rdfString,
         }],
         publisher: [_shareOrganizationField()],
+        qualifiedAttribution: [{
+            agent: [_sharePersonField()],
+            hadRole: [
+                {
+                    '@id': 'https://schema.org/author',
```

Comment:
as mentioned above, role `@id` will be `osf:admin-contributor`, `osf:write-contributor`, or `osf:readwrite-contributor` (as proposed -- can change if we agree so)

Reply:
I'm good with `@id: osf:<role>-contributor` here. Should we expect only the shorthands for this id value or could we also anticipate the full URI like `https://osf.io/vocab/...`?

Reply:
hmm, i was about to say all `https://osf.io/vocab/...` iris automatically get the `osf:...` shorthand (and that we could add an explicit shorthand if we want something else without the `osf:` prefix) -- turns out that's true in some cases, but not this one!

...i'd consider that a bug, but it's a one-line fix and i've added it to the ["add shorthands"](https://openscience.atlassian.net/browse/ENG-6190) ticket

Reply:
(so yes, expect `"@id": "osf:foo-contributor"`... soon)

Reply:
Awesome, thank you for investigating !